Doc/library/curses.rst

```diff
@@ -562,13 +572,13 @@ The module :mod:`curses` defines the following functions:
 .. function:: tigetstr(capname)
 
    Return the value of the string capability corresponding to the terminfo
-   capability name *capname*.  ``None`` is returned if *capname* is not a string
+   capability name *capname* as a byte string.  ``None`` is returned if *capname* is not a string
    capability, or is canceled or absent from the terminal description.
```

Comment:
I think we should rewrite this as `...*capname* as a bytes object.  ``None is returned if *capname* is not a terminfo "string capability", or is...`